tempto-runner/src/main/java/io/prestodb/tempto/runner/TemptoRunner.java

```diff
@@ -38,10 +37,6 @@
 
 public class TemptoRunner
 {
-    static {
-        TemptoNameServiceDescriptor.enableHostMapping();
```

Comment:
Again, just for my understanding, do we not require this anymore? Does it happen automatically now?